ui/lib/core/addon/components/enable-input.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface Args {
+  attr: Attr;
+}
```

Comment:
I think it would be nice if it wasn't dependant on a model attribute so it could be used outside of a model context. I see that the `ReadonlyFormField` component needs it but it would be nice if `attr` was optional and the component defaulted to showing an input with the stars as values. There have been recent cases where small forms aren't tied to a model so it may increase the usability of the component.

Reply:
Yes that's kind of what I was thinking. It might look like this:
```hbs
<label>Not using FormField</label>
<EnableInput>
  <input value={{this.someValue}} />
</EnableInput>
```
This would not be dependant on having a model or even mocking the attribute. You basically just get the toggle functionality and the consistent presentation of the readonly input with the star values.